ChangeLog

```diff
@@ -28,6 +28,9 @@ Bugfix
      In the context of SSL, this resulted in handshake failure. Reported by
      daniel in the Mbed TLS forum. #1351
    * Fix Windows x64 builds with the included mbedTLS.sln file. #1347
+   * Improve the robustness of mbedtls_rsa_rsassa_pkcs1_v15_verify().
```

Comment:
How _suppression of false positive_ could be an _improvement in robustness_?

Reply:
IMO this is not a bugfix and not a robustness improvement. The ChangeLog entry should be under "Changes" and should only mention silencing a false positive.

Also, I don't think this is the right way to silence that false positive. The root cause of the issue is that clang-SA assumes it's possible (1) that the `return( MBEDTLS_ERR_RSA_PUBLIC_FAILED + ret );` path is taken in `rsa_public()`and at the same time (2) that `ret == 0` on the line right after it returns. This is clearly not possible, as `MBEDTLS_ERR_RSA_PUBLIC_FAILED + ret` is non-zero for any possible return value of `mbedtls_mpi_read_binary()`. If we want to do something about it, I think it would be better to change how we return in `mbedtls_rsa_public()` to make it crystal clear that we will never return non-zero if we didn't fill the buffer.